File: LCM and HCF/lcmgcd.c
void main()
{
    int x,y,i,g=0;
    printf("Enter 2 number ");
    scanf("%d %d",&x,&y);
    for(i=0;i<=x && i<=y;i++)
    {
        if(x%i==0 && y%i==0)
        {
            g=i;
        }
    }
    printf("Gdc= %d",g);
}

File: LCM and HCF/LcmGcd.java
import java.util.Scanner;
class LcmGcd{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter 1st number : ");
        int n1=sc.nextInt();
        System.out.print("Enter 2nd number : ");
        int n2=sc.nextInt();
        sc.close();
        int num=n2,den=n1;
        int rem=num%den;
        while (rem != 0){
            num   = den;
            den = rem;
            rem   = num % den;
        }
        int gcd = den;
        int lcm = n1 * n2 / gcd;
        System.out.println("GCD of "+n1+" and  "+n2+" : "+gcd);
        System.out.println("LCM of "+n1+" and  "+n2+" : "+lcm);
    }
}